java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -451,20 +449,31 @@ class ReferenceModeStore private constructor(
         return@fn true
     }
 
+    // The launched flow cleans up backingStore entries whenever the receive queue becomes empty.
+    // This method is called by the `create` constructor, and so cleanup will occur on the
```

Comment:
this comment does not reflect the current state.

Reply:
Thanks, made a few doc updates.